src/mlang/backend_ir/bir.ml

```diff
@@ -97,6 +97,8 @@ let squish_statements (program : program) (threshold : int)
                 give_pos (SConditional (expr, t_curr_list, f_curr_list))
               in
               (f_rules, cond :: curr_stmts)
+          | SFunctionCall (f, _) ->
+              browse_bir (FunctionMap.find f program.mpp_functions) [] [] rules
```

Comment:
Here you replace the function call by its (sliced) body, that's not the purpose of this transformation.